scripts/ci-cdk-deploy.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+cdk_app_dir="cdk-app" # it is expected that the CDK app in a repository is found from this folder
+dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+# shellcheck source=.
+. "$dir"/ci-safeguard.sh
+
+if [[ -z ${ENV} ]]; then
+  echo "Missing required env ENV"
+  exit 1
+fi
+
+if [[ $(cat /tmp/is_deploy_flag 2>/dev/null) != "1" ]]; then
+  echo "Skipping deploy as FORCE_DEPLOY is not set and branch isn't develop or main"
+  exit 0
+fi
+
+if [[ ! -d "./${cdk_app_dir}" ]]; then
+  echo "Could not find CDK app folder in the repo. You should have a folder called '${cdk_app_dir}' in the root directory of your repo."
+  exit 1
+fi
+
+if [[ ! -f "${cdk_app_dir}/package.json" ]]; then
```

Comment:
Because the code below uses `npm ci` instead of `npm I`, would it be better to check the existence of both `package.json` and `package-lock.json`?

Reply:
Yes definitely good idea